scripts/invokeJestWithBucklescriptPaths.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+// changes path `.ml` or `.mli` to `.bs.js`
+// ex: foo.mli -> foo.bs.js
+
+// 1/ change ext (check)
+// 2/ run `jest`
+
+const path = require('path')
+const {spawnSync} = require('child_process')
+
+// change ext
+
+const pathArr = []
+
+process.argv.forEach(val => {
+   pathArr.push(val)
+})
+
+const newPathArr = pathArr.slice(2)
```

Comment:
You don't actually need to map over the original array to manually construct a new, second array — `Array.prototype.slice()` creates a new array!

Try something like this:

```diff
-const pathArr = []

-process.argv.forEach(val => {
-   pathArr.push(val)
-})

-const newPathArr = pathArr.slice(2)
+const newPathArr = process.argv.slice(2)
```